docs/pbc/all/ratings-reviews/202204.0/third-party-integrations/integrate-bazaarvoice.md

```diff
@@ -8,15 +8,16 @@ template: howto-guide-template
 
 The BazaarVoice app requires the following Spryker modules:
 
-* `spryker/asset: ^1.2.0`
+* `spryker/asset: ^1.3.0`
 * `spryker/asset-storage: ^1.1.0`
-* `spryker/message-broker: ^1.0.0`
-* `spryker/message-broker-aws: ^1.0.0`
-* `spryker/message-broker-extension: ^1.0.0`
+* `spryker/message-broker: ^1.3.0`
+* `spryker/message-broker-aws: ^1.3.2`
+* `spryker/message-broker-extension: ^1.1.0`
 * `spryker-shop/asset-widget: ^1.0.0`
+* `spryker-shop/cart-page: ^3.32.0`
 * `spryker-shop/product-detail-page: ^3.17.0`
 * `spryker-shop/product-category-widget: ^1.6.0`
-* `spryker-shop/shop-ui: ^1.59.0`
+* `spryker-shop/shop-ui: ^1.62.0`
 
```

Comment:
There is new [release](https://release.spryker.com/release/pull-request/2329?type=3) for BazaarVoice in release queue right now so I suggest you add these:
- `spryker-shop/checkout-page: ^3.23.0`
- `spryker-shop/merchant-page: ^1.1.0`
- `spryker-shop/merchant-profile-widget: ^1.1.0`
- `spryker-shop/merchant-widget: ^1.3.0`
- `spryker-shop/payment-page: ^1.3.0`

Reply:
fixed here https://github.com/spryker/spryker-docs/pull/1566/files